python/cudf/cudf/core/indexing.py

```diff
@@ -196,8 +197,12 @@ def _can_downcast_to_series(self, df, arg):
             if type(arg[0]) is slice:
                 if not is_scalar(arg[1]):
                     return False
-            elif pd.api.types.is_list_like(arg[0]) and (
-                pd.api.types.is_list_like(arg[1]) or type(arg[1]) is slice
+            elif (
+                pd.api.types.is_list_like(arg[0]) or is_column_like(arg[0])
+            ) and (
+                pd.api.types.is_list_like(arg[1])
```

Comment:
We have our own implementation of `is_list_like`: https://github.com/rapidsai/cudf/blob/19a1d7965afecf8e17d6c3323e02a7d4f7e0eb20/python/cudf/cudf/utils/dtypes.py#L211-L229